certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "dates": {
+          "description": "Gets or Sets Dates",
+          "type": "array",
+          "items": {
+            "format": "date-time",
+            "type": "string"
+          }
+        },
+        "productReferenceKey": {
+          "format": "uuid",
+          "description": "Gets or Sets ProductReferenceKey",
+          "type": "string"
+        },
+        "submissionReference": {
+          "description": "Gets or Sets SubmissionReference",
+          "type": "string"
+        },
+        "code": {
+          "description": "Gets or Sets Code",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or Sets Description",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderAddressType": {    
+      "description": "Gets or Sets TitleOrderAddressType",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.County": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or Sets Name",
+          "type": "string"
+        },
+        "fips": {
+          "description": "Gets or Sets Fips",
+          "type": "string"
+        },
+        "statecode": {
+          "format": "int64",
+          "description": "Gets or Sets Statecode",
+          "type": "integer"
+        },
+        "countycode": {
+          "format": "int64",
+          "description": "Gets or Sets Countycode",
+          "type": "integer"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderAddress": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "full": {
+          "description": "Gets or Sets Full",
+          "type": "string"
+        },
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddressType"
+        },
+        "line1": {
+          "description": "Gets or Sets Line1",
+          "type": "string"
+        },
+        "line2": {
+          "description": "Gets or Sets Line2",
+          "type": "string"
+        },
+        "city": {
+          "description": "Gets or Sets City",
+          "type": "string"
+        },
+        "state": {
+          "description": "Gets or Sets State",
+          "type": "string"
+        },
+        "zip": {
+          "description": "Gets or Sets Zip",
+          "type": "string"
+        },
+        "county": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.County"
+        },
+        "country": {
+          "description": "Gets or Sets Country",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.CommunicationType": {     
+      "description": "Gets or Sets CommunicationType",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.Communication": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.